Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -187,7 +187,12 @@ If not specified, the default SDN will be used.
 Configure the network that will be enabled for the cluster and services.
 
 * `provider`: Specifies the network provider that will be used to connect the network interface. You can choose between `host`, and `multus`.
-* `selectors`: List the network selector(s) that will be used associated by a key.
```

Comment:
How about also updating [ceph-configuration.md](https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-configuration/#osd-dedicated-network) to point to the new way of applying public/cluster settings?

Reply:
I had forgotten this doc section was written only as an OSD topic. I think the relevant parts of that doc are actually present in the "Ceph public and cluster networks" section that is now below, here in the CRD doc. I think it would be good to consolidate the information somewhere, and my current thought is to remove the section of the advanced config doc you linked.

Alternately, I could rewrite the Advanced config, "OSD Dedicated Network" section to contain the fuller info about public/cluster networks that I have in the CRD doc below currently. I'd then then link to the advanced doc from here, with brief info as needed. The struggle with this may be that information about `selectors` and `addressRanges` will predominantly be in a different doc, which could be confusing.

Do you have thoughts about either approach?

Reply:
What if we leave that section in the advanced config untouched, except put a note at the top that says it is recommended to follow the new approach, and add a link to the new approach? I hesitate to remove the section completely since it's still valid if someone wants to use the ceph config override.

Reply:
Done, and also added a paragraph about changing network configs only with care since this section contains something similar.